src/GraphQL/Execution/Nodes/ArrayExecutionNode.cs

```diff
@@ -56,6 +59,38 @@ public override object ToValue()
             return items;
         }
 
+        public override bool ClearErrorNodes()
+        {
+            if (Items == null)
+                return true;
+
+            if (Items.Count == 0)
+                return false;
+
+            var ok = false;
```

Comment:
?

Reply:
Short circuit so the type doesn’t have to be checked for every iteration

Reply:
Note that ClearErrorNodes still needs to be called for the descendants so can’t just return false.

Reply:
Please give a better name to this variable.